__tests__/app.test.js

```diff
@@ -485,3 +485,60 @@ describe("GET /api/articles (querying with topic)", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id (comment_count)", () => {
+  test("200: Responds with an article object with a positive comment_count", () => {
+    return request(app)
+      .get("/api/article/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          comment_count: 11
+        });
+      });
+  });
```

Comment:
This is the same endpoint as a previous task, and we are already asserting all of the other properties so this test should only need to assert that the comment_count property has been added